Regression/WarpX-tests.ini

```diff
@@ -167,6 +167,22 @@ compileTest = 0
 doVis = 0
 analysisRoutine = Examples/Tests/pml/analysis_pml_psatd_rz.py
 
+[pml_x_galilean]
+buildDir = .
+inputFile = Examples/Tests/pml/inputs_2d
+runtime_params = algo.maxwell_solver=psatd psatd.update_with_rho=1 warpx.do_dynamic_scheduling=0 diag1.fields_to_plot=Ex Ey Ez Bx By Bz rho divE warpx.cfl=0.7071067811865475 warpx.do_pml_dive_cleaning=0 warpx.do_pml_divb_cleaning=0 psatd.current_correction=0 warpx.abort_on_warning_threshold=medium psatd.v_galilean=0. 0. 0.99498743710662
```

Comment:
@RemiLehe How was the Galilean velocity chosen to be `psatd.v_galilean=0. 0. 0.99498743710662` in this case, when you first implemented this PR? Or in other words, how is the velocity supposed to be computed in a test like this one, where we have only one laser and nothing else (no plasma particles, no boost)?

Reply:
That's a good question. I think that, in this PR, the Galilean velocity was chosen arbitrarily.